### PR TITLE
Updated meta details for fgam and updated audit logging docs

### PR DESCRIPTION
**Pull request by dsieczko**

### CODE REVIEW

File: docs/spicedb-dedicated/audit-logging.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+
 # Audit Logging in SpiceDB Dedicated
 
-:::note
-Audit Logging is still an in-preview feature, which means it does not have full UI for configuration
-:::
+Audit Logging allows you to capture a log of all API calls made to SpiceDB, and is available in SpiceDB Enterprise, the licensable binary we use to power SpiceDB Dedicated.
 
-Audit Logging is an optional feature for logging of all API made operations to a Permissions System, when deployed with SpiceDB Dedicated and SpiceDB Enterprise.
+When enabled and properly configured, SpiceDB Enterprise will asynchronously log every API call made to it and emit a stream to your preferred log sink. The logs contain full details related to a request, including the API token, RPC, payload, request IP, and response.
```
Comment (vroldanbet) [inline docs/spicedb-dedicated/audit-logging.md:11]: we don't log the tokens - that'd be scary 😄 

```suggestion
When enabled and properly configured, SpiceDB Enterprise will asynchronously log every API call made to it and emit a stream to your preferred log sink. The logs contain full details related to a request, including a hash of the API token, RPC, payload, request IP, response and any possible errors.
```

**Review by vroldanbet — APPROVED** [review]

File: docs/spicedb-dedicated/fgam.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+---
+title: Fine-Grained Access Management for SpiceDB Dedicated
+description: Learn how to configure Fine Grained Access Management in SpiceDB Dedicated.
```
Comment (samkim) [inline docs/spicedb-dedicated/fgam.md:3]:
```suggestion
description: Learn how to configure Fine-Grained Access Management in SpiceDB Dedicated.
```

File: docs/spicedb-enterprise/fgam.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+---
+title: Fine-Grained Access Management for SpiceDB Enterprise
+description: Learn how to configure Fine Grained Access Management for SpiceDB Enterprise.
```
Comment (samkim) [inline docs/spicedb-enterprise/fgam.md:3]:
```suggestion
description: Learn how to configure Fine-Grained Access Management for SpiceDB Enterprise.
```

File: docs/spicedb-dedicated/audit-logging.md (hunk 1 above)
Comment (dsieczko) [reply docs/spicedb-dedicated/audit-logging.md:11]: Thanks @vroldanbet !

File: docs/spicedb-enterprise/fgam.md (hunk 3 above)
Comment (dsieczko) [reply docs/spicedb-enterprise/fgam.md:3]: Good catch! Thanks @samkim

**Comment by samkim** [comment]
@dsieczko need to fix these formatting issues and this should be good to go https://github.com/authzed/docs/actions/runs/6371591800/job/17293665421?pr=162#step:6:7

**Comment by dsieczko** [comment]
Hey @samkim  - I fixed the newline (my bad) but the other one is I think the linter misunderstanding the Docusaurus top of file meta page settings syntax. It complains about "Multiple top-level headings in the same document" but there is only one # in the doc

**Pull request merged**